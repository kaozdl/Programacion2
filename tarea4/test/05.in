# cola_binarios
encolar
desencolar 1
insertar_binario (3,x) (1,y) (5,z) (2,m) (8,c) (-2,b) (12,a) (9,q) .
encolar
desencolar 1
desencolar 5
desencolar 1
desencolar 3
Fin
