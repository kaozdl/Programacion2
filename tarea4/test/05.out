1>
2>3>4>
--(5,z)
-(1,y)
(3,x)
--(9,q)
-(2,m)
--(8,c)
---(-2,b)
----(12,a)
4
2
5>6>
(12,a)
7>
--(5,z)
-(1,y)
(3,x)
--(9,q)
-(2,m)
--(8,c)
---(-2,b)
----(12,a)
8>
-(5,z)
(1,y)
9>10>Fin.
